xbmc/Util.cpp

```diff
@@ -1754,7 +1754,7 @@ void CUtil::ScanForExternalSubtitles(const std::string& strMovie, std::vector<st
   CFileItemList items;
   vector<std::string> strLookInPaths;
 
-  int flags = DIR_FLAG_NO_FILE_DIRS | DIR_FLAG_READ_CACHE | DIR_FLAG_NO_FILE_INFO;
+  int flags = DIR_FLAG_NO_FILE_DIRS | DIR_FLAG_BYPASS_CACHE | DIR_FLAG_NO_FILE_INFO;
 
```

Comment:
DIR_FLAG_BYPASS_CACHE will not update the dir-cache, for which there is no reason. Just drop the DIR_FLAG_READ_CACHE and all should be well.